inference-engine/src/mkldnn_plugin/nodes/mkldnn_input_node.h

```diff
@@ -21,19 +22,16 @@ class MKLDNNInputNode : public MKLDNNNode {
     void createPrimitive() override;
     bool created() const override;
 
-    void execute(mkldnn::stream strm) override;
-    void withMeanImage() {
-        isMeanImage = true;
-    }
+    void withMeanImage();
+    MKLDNNMemoryCPtr getMemoryPtr() const;
 
-    const InferenceEngine::Blob::CPtr getConstBlob() const {
-        return constBlob;
-    }
+private:
```

Comment:
minor: two private sections

Reply:
I use private sections to separate function declarations and class members. (: